### ISSUE TITLE
Option to only show "test regex..." only when cursor is on line.

### ISSUE THREAD

**Issue by Tain101** [chrmarti/vscode-regex#61]
If you have several lines of regex, the "test regex..." line creates a lot of wasted space. An option to only display the text for one regex/line at a time would solve the issue.

**Comment by Emasoft** [comment]
I agree. It should also recognize the regex strings so it will only display "Test Regex..." on the appropriate lines of code. What is the regex to match a regex string, BTW? Here I found a regex to match Javascript regex strings:
https://stackoverflow.com/questions/17843691/javascript-regex-to-match-a-regex
But for Python or other languages there are some differences.
